build.gradle

```diff
@@ -66,7 +66,6 @@ kotlin {
                 implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.8'
                 implementation 'com.fasterxml.jackson.module:jackson-module-kotlin:2.9.8'
                 implementation 'com.fasterxml.jackson.module:jackson-module-parameter-names:2.9.8'
-                implementation "org.jetbrains.kotlinx:kotlinx-io-jvm:$kotlinx_io_version"
```

Comment:
Shouldn't that line remain? We would get problems under non-MacOS platforms, no?

Reply:
yes it should remain. I'll put it back